src/e3/os/platform.py

```diff
@@ -381,10 +383,31 @@ def get(
         # default (cross case)
         if version == UNKNOWN:
             if is_host:
+                SystemInfo.fetch_system_data()
                 version, kernel_version = SystemInfo.os_version()
+                if not version_minor:
```

Comment:
So we won't enter here if `version_minor` is `UNKNOWN`

Reply:
Indeed, I will make the change.

Reply:
Done.